packages/web/docs/src/components/ecosystem-management.tsx

```diff
@@ -284,16 +323,50 @@ function EdgeLabel(props: EdgeLabelProps) {
 interface NodeProps extends React.HTMLAttributes<HTMLElement> {
   title: string;
   description?: ReactNode;
-  highlighted: boolean;
+  edges: number[];
+  highlightedEdge: number | null;
+  onHighlight: (edges: number[]) => void;
 }
-function Node({ title, description, children, highlighted, className, ...rest }: NodeProps) {
+function Node({
+  title,
+  description,
+  children,
+  edges,
+  highlightedEdge,
+  className,
+  onHighlight,
+  ...rest
+}: NodeProps) {
+  const highlighted = edges.includes(highlightedEdge!);
+
+  const hovered = useRef(false);
+
   return (
     <div
+      onPointerOver={event => {
+        if (hovered.current || event.currentTarget !== event.target) {
+          return;
+        }
+
+        hovered.current = true;
+
+        if (edges.includes(highlightedEdge!)) return;
+        onHighlight([edges[0]]);
+      }}
+      onPointerOut={event => {
+        if (
+          !event.currentTarget.contains(event.relatedTarget as Node) &&
```

Comment:
onPointerOut and onPointerOver trigger all the time _inside_ of the chart Node, so we need to check if the `relatedTarget` is inside of `currentTarget`.